CHANGELOG.md

```diff
@@ -2,6 +2,19 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+### [2.28.7](https://github.com/serverless/serverless/compare/v2.28.6...v2.28.7) (2021-03-04)
+
+### Maintenance Improvements
```

Comment:
I think it should be categorized as  bug fix and put into _Packaging_ category, we probably should make this item more manual, stating something as:

_Fix packaging performance regression and increased number of observed EMFILE crashes, by reverting intensive `bluebird` related refactors as listed below:_

(Users may observe packaging regression, and just stating that we made some maintenance improvements doesn't give a hint, that it could be a fix for that)

Reply:
good idea - I've pushed an adjusted changelog